frontend/src/components/associations/marketplace/counter/OrderSummary.tsx

```diff
@@ -1,61 +1,72 @@
+                onClick={makeOrder}
+            >
+                <span className="fe fe-shopping-cart" /> Passer la commande
+            </Button>
+            <strong>
+                Total :{" "}
+                {Object.keys(basket)
+                    .map((key) => basket[key])
+                    .map(({ product, quantity }) => product.price * quantity)
+                    .reduce((acc, val) => acc + val, 0)}
+                €
+            </strong>
+        </Card.Footer>
+    </Card>
+);
 
-    return (
-        <Card className={"mt-5"}>
-            <Card.Header>
-                <Card.Title>Commande</Card.Title>
-            </Card.Header>
-            <div className="table-responsive">
-                <div className="dataTables_wrapper no-footer">
-                    <table
-                        className="table card-table table-vcenter datatable dataTable no-footer table-striped"
-                        role="grid"
-                    >
-                        <tbody>
-                            {Object.keys(basket)
-                                .map((key) => basket[key])
-                                .map(({ product, quantity, status }) => (
-                                    <OrderStatus
-                                        key={product.id}
-                                        product={product}
-                                        quantity={quantity}
-                                        status={status}
-                                        decreaseNumber={() =>
-                                            removeFromBasket(product)
-                                        }
-                                    />
-                                ))}
-                        </tbody>
-                    </table>
-                </div>
-            </div>
-            <Card.Footer>
-                <Button
-                    variant={"success"}
-                    size={"sm"}
-                    className="float-right"
-                    onClick={makeOrder}
-                >
-                    <span className="fe fe-shopping-cart" /> Passer la commande
-                </Button>
-                <strong>
-                    Total :{" "}
-                    {Object.keys(basket)
-                        .map((key) => basket[key])
-                        .map(
-                            ({ product, quantity }) => product.price * quantity
-                        )
-                        .reduce((acc, val) => acc + val, 0)}
-                    €
-                </strong>
-            </Card.Footer>
-        </Card>
-    );
-};
+const columns = (decreaseNumber) => [
+    {
+        key: "product",
+        header: "Produit",
+        render: ({ product, quantity, status }) => (
+            <>
+                {product.name} <span className="text-muted">x{quantity}</span>
+            </>
+        ),
+    },
+    {
+        key: "value",
+        header: "Valeur",
```

Comment:
![image](https://user-images.githubusercontent.com/11032610/83331712-0b81b500-a298-11ea-8b6e-194f6d0eda7a.png)

I think « Montant » would be better.

There is also twice the quantity information (x2). Put a space between the « x » and the « 2 »